ansible_ai_connect_chatbot/src/App.test.tsx

```diff
@@ -152,6 +246,16 @@ test("Basic chatbot interaction", async () => {
     .toBeVisible();
   await expect.element(view.getByText("Create variables")).toBeVisible();
 
+  const copyIcon = await screen.findByRole("button", {
```

Comment:
Although this is not related to the streaming support, test cases for the copy button support was added for better code coverage.

Reply:
thank you @TamiTakamiya